src/utils/utils.ts

```diff
@@ -483,6 +485,19 @@ export function getSourceFromLocation(source: string, location: SourceLocation):
   return [...previousLines, ...currentLine, ...followingLines].join('\n');
 }
 
+export function callClassHashScript(filePath: string): string {
+  const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', 'warp_venv', 'bin')}:$PATH`;
+  const classHash = execSync(
+    `${warpVenvPrefix} python starknet-scripts/compute_class_hash.py ${filePath}`,
```

Comment:
The path to starknet-scripts folder here needs to be relative to the current file, so that it works properly when its distributed through a npm package, similar to how `warpVenvPrefix` is done